README.md

```diff
@@ -1,70 +1,16 @@
-| [SA102](mapping/sa102_mapping_v2.csv)<br />Employment        | [individuals-expenses-api](https://developer.service.hmrc.gov.uk/api-documentation/docs/api/service/individuals-expenses-api) <br />[individuals-employments-income-api](https://developer.service.hmrc.gov.uk/api-documentation/docs/api/service/individuals-employments-income-api) |
-| [SA103F](mapping/sa103f_mapping_v1.csv)<br />Self-employment (Full) | [cis-deductions-api](https://developer.service.hmrc.gov.uk/api-documentation/docs/api/service/cis-deductions-api) <br />[individual-losses-api](https://developer.service.hmrc.gov.uk/api-documentation/docs/api/service/individual-losses-api) <br />[individuals-disclosures-api](https://developer.service.hmrc.gov.uk/api-documentation/docs/api/service/individuals-disclosures-api) <br />[self-employment-business-api](https://developer.service.hmrc.gov.uk/api-documentation/docs/api/service/self-employment-business-api) |
-| [SA105](mapping/sa105_mapping_v1.csv)<br />UK Property       | [individual-losses-api](https://developer.service.hmrc.gov.uk/api-documentation/docs/api/service/individual-losses-api) <br />[property-business-api](https://developer.service.hmrc.gov.uk/api-documentation/docs/api/service/property-business-api) |
-| [SA106](mapping/sa106_mapping_v2.csv)<br />Foreign           | [individual-losses-api](https://developer.service.hmrc.gov.uk/api-documentation/docs/api/service/individual-losses-api) <br />[individuals-dividends-income-api](https://developer.service.hmrc.gov.uk/api-documentation/docs/api/service/individuals-dividends-income-api) <br />[individuals-insurance-policies-income-api](https://developer.service.hmrc.gov.uk/api-documentation/docs/api/service/individuals-insurance-policies-income-api) <br />[individuals-other-income-api](https://developer.service.hmrc.gov.uk/api-documentation/docs/api/service/individuals-other-income-api) <br />[individuals-pensions-income-api](https://developer.service.hmrc.gov.uk/api-documentation/docs/api/service/individuals-pensions-income-api) <br />[individuals-reliefs-api](https://developer.service.hmrc.gov.uk/api-documentation/docs/api/service/individuals-reliefs-api) <br />[individuals-savings-income-api](https://developer.service.hmrc.gov.uk/api-documentation/docs/api/service/individuals-savings-income-api) <br />[property-business-api](https://developer.service.hmrc.gov.uk/api-documentation/docs/api/service/property-business-api) |
-| [SA108](mapping/sa108_mapping_v1.csv)<br />Capital Gains Summary | [individuals-capital-gains-income-api](https://developer.service.hmrc.gov.uk/api-documentation/docs/api/service/individuals-capital-gains-income-api) |
+Many parameters in some Income Tax (Making Tax Digital) APIs map to box numbers in <a href="https://www.gov.uk/self-assessment-tax-return-forms">Self Assessment tax return forms (GOV.UK)</a>. For more information, see [Mappping CSV files](mapping/mapping-csv-files.md).
```

Comment:
Maybe just "Parameters in some Income Tax"... etc

Reply:
OK. Although Dale and Rajar spent a lot of time considering that sentence.